### ISSUE TITLE
Error if no shipping method found

### ISSUE THREAD

**Issue by NyanKiyoshi** [mirumee/saleor-storefront#107]
If we try to order the product `63` (https://pwa.getsaleor.com/product/red-dwarf-red-paint/63/) (don't add any other products, otherwise it seems to work) from France, it should raise this error if you submit nothing (because of `shippingMethodId: ""`):

```
{
    "data": {
        "checkoutShippingMethodUpdate": {
            "errors": [{
                "field": "shippingMethodId",
                "message": "Couldn't resolve to a node: ",
                "__typename": "Error"
            }],
            "checkout": null,
            "__typename": "CheckoutShippingMethodUpdate"
        }
    }
}
```

## Request
```
fragment Checkout on Checkout {
    token id totalPrice {
        net {
            amount __typename
        }
        gross {
            amount localized __typename
        }
        currency __typename
    }
    subtotalPrice {
        net {
            amount __typename
        }
        gross {
            amount localized __typename
        }
        currency __typename
    }
    billingAddress {
        firstName lastName companyName streetAddress1 streetAddress2 city postalCode country {
            code country __typename
        }
        countryArea phone __typename
    }
    shippingAddress {
        firstName lastName companyName streetAddress1 streetAddress2 city postalCode country {
            code country __typename
        }
        countryArea phone __typename
    }
    email availableShippingMethods {
        id name price {
            currency amount localized __typename
        }
        __typename
    }
    shippingMethod {
        id name price {
            currency amount localized __typename
        }
        __typename
    }
    shippingPrice {
        net {
            amount __typename
        }
        gross {
            amount localized __typename
        }
        currency __typename
    }
    lines {
        id quantity totalPrice {
            net {
                amount __typename
            }
            gross {
                amount localized __typename
            }
            currency __typename
        }
        variant {
            id name price {
                amount currency localized __typename
            }
            product {
                id name thumbnailUrl thumbnailUrl2x: thumbnailUrl(size: 510) __typename
            }
            __typename
        }
        quantity __typename
    }
    __typename
}
mutation updateCheckoutShippingOptions($checkoutId: ID!, $shippingMethodId: ID!) {
    checkoutShippingMethodUpdate(checkoutId: $checkoutId, shippingMethodId: $shippingMethodId) {
        errors {
            field message __typename
        }
        checkout { ...Checkout __typename
        }
        __typename
    }
}

{
"operationName":"updateCheckoutShippingOptions",
"variables":{"checkoutId":"Q2hlY2tvdXQ6NTdkMDY1MDUtYzQ4Ny00YjA2LTkyYjQtM2M3MGI3OWYyZDky","shippingMethodId":""
}
```

**Comment by maarcingebala** [comment]
Not sure if this is related but there is also problem when API returns empty available shipping methods:
![image](https://user-images.githubusercontent.com/5421321/48617014-8bd21500-e995-11e8-8992-c1a4ef109661.png)

In such case we should display a message and disable navigation to further checkout steps. We can reuse the message from the old dashboard:
"Unfortunately we do not ship to your selected country. 
Please enter an alternative shipping address or contact us, if you think that's a mistake."

**Comment by maarcingebala** [comment]
E.g. response with empty shipping methods which fails in UI:
```
"checkoutEmailUpdate": {
      "errors": [],
      "checkout": {
        "availableShippingMethods": [],
        "shippingMethod": null
      }
    }
```

**Comment by piotrgrundas** [comment]
Fixed in #205
